pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
+
+- Execute a command (using the command module) on a group of hosts:
+
+`ansible {{group}} --module-name command --args '{{my command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} --become --ask-become-pass --module-name command --args '{{my command}}'`
+
+- Execute a command using a custom inventory file:
+
+`ansible {{group}} --inventory-file {{path/to/inventory/file}} --module-name command --args '{{my command}}'`
```

Comment:
Here the short options would be even more appropriate, since the command is quite long with the expanded option names. Perhaps we could introduce the options using the expanded versions, and from then on use the short ones?